test/test_helper.rb

```diff
@@ -42,3 +42,18 @@ class TestCase
     # Add more helper methods to be used by all tests here...
   end
 end
+
+# Use a memory-backed `Rails.cache` for tests and clear it before each test to avoid cross-test
+# leakage. This simplifies tests and emoves the need to inject a cache store into most specs, but
+# a shared cache can still cause uses with parallel fetching, as we use in the Merged Search
+# Service.
+#
+# If you hit flaky tests related to cache  access under concurrency, consider injecting a per-test
+# cache or avoiding shared mutable state in the threaded path.
+Rails.cache = ActiveSupport::Cache::MemoryStore.new
```

Comment:
It might be better to change this value in the config rather than overriding it here?

https://github.com/MITLibraries/timdex-ui/blob/main/config/environments/test.rb#L26